playground/tailwind.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   theme: {
     extend: {
       fontFamily: {
-        'sans': ['"Noto Sans"', ...defaultTheme.fontFamily.sans],
+        'sans': ['Arial, sans-serif', ...defaultTheme.fontFamily.sans],
       },
```

Comment:
If we remove, this entire tidbit can be removed to preserve system font stack

Reply:
Your suggestion in the issue about bundling the font is great, so I have updated the title to reflect this. Feel free to update this PR, or create a new PR, whatever is easier.